otcextensions/sdk/auto_scaling/v1/_proxy.py

```diff
@@ -10,6 +10,9 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 from openstack import proxy
+from openstack import utils
+from openstack import resource
```

Comment:
sadly our pep job doesn't catch this, but those imports must be sorted alphabetically

Reply:
Done